check-node.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+
+import json, subprocess, re, sys
+
+current_node_version = []
+expected_node_version = []
+
+def get_version(s):
+	return re.sub('[v\n]', '', s).split('.')
+
+with open('package.json') as f:
+	data = json.load(f)
+expected_node_version = get_version(str(data['engines']['node']))
+
+try:
+	current_node_version = get_version(subprocess.check_output('node --version', shell=True))
+except subprocess.CalledProcessError as e:
+	print 'Could not get node version, please install node with version %s' % expected_node_version
+	print e.output
+	sys.exit(1)
+
+if len(current_node_version) < 2 or len(expected_node_version) < 2:
+	print 'Not enough information about node version'
+	sys.exit(1)
+
+# Major and minor version are expected to by equal.
+if current_node_version[0] != expected_node_version[0] or current_node_version[1] != expected_node_version[1]:
+	print 'Current node version is %s but should be %s. Please install nvm and execute nvm use.' % (current_node_version, expected_node_version)
```

Comment:
currently message on console is:
_Curent node version is ['10', '15', '3'] but should be ['6', '16']._
Can we improve message? e.g. 
_Curent node version is 10.15 but should be 6.16_

Reply:
Fixed, now it prints:
`Current node version is 10.15.3 but should be 6.16. Please install nvm and execute nvm use.`